.github/styles/vocab.txt

```diff
@@ -24,7 +24,7 @@ backported
 backporting
 Bigtable
 Billett
-Bitbucket
+bitbucket
```

Comment:
vale again

Reply:
Isn't `Bitbucket` the correct version though? :-) Couldn't spot any violations (anymore).

Reply:
It is - but vale started checking api-report.md files too, where the (lowercase) package name is seen as plain text. I've started to kind of get annoyed with vale being less than helpful so I reverted to putting the any-case versions in here just to silence it, and then try to be vigilant while reading PRs manually instead.